Building Docs Breaks on Python 3.10

Attempting to build the docs with Python 3.10, which is now the default in Kali Linux causes an error:
```
$ git clone https://github.com/Gallopsled/pwntools.git
$ cd pwntools/
$ cd docs
$ python3 -m venv ./venv
$ source ./venv/bin/activate
$ python3 -m pip install -q pwntools
$ python3 -m pip install -r requirements.txt 
...
$ make html
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v4.5.0

Extension error:
Could not import extension sphinxcontrib.napoleon (exception: cannot import name 'Callable' from 'collections' (/usr/lib/python3.10/collections/__init__.py))
make: *** [Makefile:53: html] Error 2
```

Other info:
```
$ pip list | grep pwntools
pwntools                      4.8.0
```
```
$ python3 --version
Python 3.10.4
```
```
$ cat /etc/os-release
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2022.2"
VERSION_ID="2022.2"
VERSION_CODENAME="kali-rolling"
ID_LIKE=debian
```

Found this in the Sphinx repo: https://github.com/sphinx-doc/sphinx/issues/10378/. If I change in `source/conf.py`: 
```
'sphinxcontrib.napoleon'
```
to
```
'sphinx.ext.napoleon'
```
it works.

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -12,8 +12,7 @@ pypandoc
 pyserial>=2.7
 requests>=2.5.1
 ropgadget>=5.3
-sphinx==1.8.6; python_version < '3.0.0'
-sphinx==4.5.0; python_version >= '3.0.0'
+sphinx==1.8.6; python_version<'3'
+sphinx>=4.5.0; python_version>='3'
 sphinx_rtd_theme
-sphinxcontrib-napoleon
 sphinxcontrib-autoprogram<=0.1.5

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -42,8 +42,8 @@ extensions = [
     'sphinx.ext.coverage',
     'sphinx.ext.todo',
     'sphinx.ext.intersphinx',
+    'sphinx.ext.napoleon',
     'sphinxcontrib.autoprogram',
-    'sphinxcontrib.napoleon'
 ]
 
 # Disable "info" logging directly to stdout by Sphinx

--- a/pwnlib/util/safeeval.py
+++ b/pwnlib/util/safeeval.py
@@ -6,6 +6,7 @@ _const_codes = [
     'BUILD_CONST_KEY_MAP', 'BUILD_STRING',
     'LOAD_CONST','RETURN_VALUE','STORE_SUBSCR', 'STORE_MAP',
     'LIST_TO_TUPLE', 'LIST_EXTEND', 'SET_UPDATE', 'DICT_UPDATE', 'DICT_MERGE',
+    'COPY', 'RESUME',
     ]
 
 _expr_codes = _const_codes + [
@@ -15,6 +16,7 @@ _expr_codes = _const_codes + [
     'BINARY_MODULO','BINARY_ADD','BINARY_SUBTRACT',
     'BINARY_LSHIFT','BINARY_RSHIFT','BINARY_AND','BINARY_XOR',
     'BINARY_OR',
+    'BINARY_OP',
     ]
 
 _values_codes = _expr_codes + ['LOAD_NAME']

